oedatamodel/latest/v100/datapackage/oedatamodel-readable_datapackage.json

```diff
@@ -0,0 +1,160 @@
+        "decimalSeparator": "."} },
+        {"profile": "tabular-data-resource",
+            "name": "oed-readable_timeseries",
+            "path": "oedatamodel-readable_timeseries.csv",
+            "format": "csv",
+            "encoding" : "UTF-8",
+            "schema": {
+                "fields": [
+                    {"name": "scalar_id", "description": "Unique identifier", "type": "bigint", "unit": null},
+                    {"name": "scenario_id", "description": "Scenario name", "type": "text", "unit": null},
+                    {"name": "region", "description": "Country or region", "type": "json", "unit": null},
+                    {"name": "input_energy_vector", "description": "", "type": "integer", "unit": null},
+                    {"name": "output_energy_vector", "description": "", "type": "text", "unit": null},
+                    {"name": "parameter_name", "description": "", "type": "text", "unit": null},
+                    {"name": "technology", "description": "", "type": "text", "unit": null},
+                    {"name": "technology_type", "description": "", "type": "text", "unit": null},
+                    {"name": "unit", "description": "Parameter unit", "type": "text", "unit": null},
+                    {"name": "timeindex start", "description": "Start timestemp", "type": "timestamp", "unit": null},
+                    {"name": "timeindex stop", "description": "Stop timestemp", "type": "timestamp", "unit": null},
+                    {"name": "timeindex resolution", "description": "Timesteps", "type": "intervall", "unit": null},
+                    {"name": "series", "description": "Timesteps", "type": "array[decimal]", "unit": null},
+                    {"name": "tags", "description": "Free classification with key-value pairs", "type": "hstore", "unit": null},
+                    {"name": "method", "description": "Method type (sum, mean, median)", "type": "json", "unit": null},
+                    {"name": "source", "description": "Source", "type": "text", "unit": null},
+                    {"name": "comment", "description": "Comment", "type": "text", "unit": null} ],
+                "primaryKey": ["scalar_id"],
+                "foreignKeys": [{
+                        "fields": ["scenario_id"],
+                        "reference": {
+                            "resource": "oed_scenario",
+                            "fields": ["scenario_id"] } } ] },
+            "dialect":
+                {"delimiter": ";",
+                "decimalSeparator": "."} } ],
+
+"review": {
+    "path": "",
+    "badge": ""},
+"metaMetadata":
+    {"metadataVersion": "OEP-1.4.0",
+    "metadataLicense":
+        {"name": "CC0-1.0",
+        "title": "Creative Commons Zero v1.0 Universal",
+        "path": "https://creativecommons.org/publicdomain/zero/1.0/"} },
+"_comment":
+    {"metadata": "Metadata documentation and explanation (https://github.com/OpenEnergyPlatform/organisation/wiki/metadata)",
```

Comment:
```suggestion
"_comment":
    {"metadata": "Metadata documentation and explanation (https://github.com/OpenEnergyPlatform/organisation/wiki/metadata)",
    "source": "Reference data source or model (modelling framework including version number) used to generate results",
```

Reply:
We have already introduced a source key in the oemetadata format.  There, used datasets can be referenced. In a new oemetadata release we could introduce further subkeys for e.g. modeling-framework and mf_version to reference frameworks there. I will create an issue for this.

Reply:
The link to the source key would not have answered my question about "where do I put the model identifier?" even if I would have had the link. 

The key question is how make it as easy to understand/find for an average/novice user...  And from reviewing the PR, this seemed the best place to put a clarification.

Reply:
Sure, my comment was meant to be a solution suggestion. IMO currently model framework references are not directly supported in the oemetadata, without a deeper understanding of the oemetadata format.  I agree with you that in order to avoid confusion, it is important that the fields are clearly defined/named and extra description must be offered to ensure better usability. I will mention this in the Issue.